apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -17,9 +25,32 @@ import { Component } from '@angular/core';
       }
     }
   `,
+  animations: [
+    trigger('slideIn', [
```

Comment:
Personally i prefer to create a component animations file `app.animation.ts` and keep the animation outside of the component, to keep the component as simple as possible (less lines of code). But that's totally fine too.